13/umbraco-ui-builder/collections/related-collections.md

```diff
@@ -0,0 +1,99 @@
+---
+description: Configuring **many-to-many** relationships in Umbraco UI Builder, the backoffice UI builder for Umbraco.
+---
+
+# Related Collections
+
+Related collections identify with a **many-to-many** relationship context, when multiple entities from a parent collection are associated with multiple entities from a different collection, using a junction entity.
+
+![Child Collection](../images/related_collections_child.png)
+
+![Parent Collection](../images/related_collections_parent.png)
+
+![Entity Picker](../images/related_collections_entity_picker.png)
+
+## Collections Representation
+
+A representation of your collections would look like this:
+
+![Related Collections Diagram](../images/related_collections_diagram.png)
+
+And the entities would be represented using the following Poco models:
+
+```csharp
+[TableName("Person")]
+[PrimaryKey("Id")]
+public class Person
+{
+    [PrimaryKeyColumn]
+    public int Id { get; set; }
+
+    public string Name { get; set; }
+
+    public string Email { get; set; }
+}
+```
+
+```csharp
+[TableName("Color")]
+[PrimaryKey("Id")]
+public class Color
+{
+    [PrimaryKeyColumn]
+    public int Id { get; set; }
+
+    public string Name { get; set; }
+}
+```
+
+```csharp
+[TableName("PersonColor")]
+[PrimaryKey(new[] { "PersonId", "ColorId" })]
+public class PersonColor
+{
+    [PrimaryKeyColumn]
+    public int PersonId { get; set; }
+
+    [PrimaryKeyColumn]
+    public int ColorId { get; set; }
+}
+```
+
+## Defining a related collection
```

Comment:
I think this section could be a bit clearer, maybe:

- Add an introductory sentence explaining how you configure a related collection in two parts - firstly by defining the relationship and then by adding a picker for the related entities to the user interface.
- Then indent the heading each section below one level so they become sub-sections under this heading.